Makefile

```diff
@@ -62,6 +62,9 @@ RBAC_ROOT ?= $(MANIFEST_ROOT)/rbac
 # Allow overriding the imagePullPolicy
 PULL_POLICY ?= Always
 
+# Always use buildkit
+DOCKER_BUILDKIT=1
```

Comment:
Why is buildkit mandatory? Can we make it default, but optional? If it is mandatory, I'd propose that we add a short comment describing why.